vendor/github.com/containers/common/libnetwork/etchosts/hosts.go

```diff
@@ -252,7 +253,8 @@ func parseExtraHosts(extraHosts []string, hostContainersInternalIP string) (Host
 			}
 			ip = hostContainersInternalIP
 		}
-		e := HostEntry{IP: ip, Names: []string{values[0]}}
```

Comment:
you cannot change vendor code directly, this must be first submitted as PR in https://github.com/containers/common then we can update the dependency here

Reply:
Yes I've seen that, that's why I just opened this PR : https://github.com/containers/common/pull/2139 

So on this repository, I should just remove my changes from hosts.go ?

Reply:
As the PR has been merged, how do we proceed to include a version of the common library into this code ? 
I've seen that it is defined there https://github.com/containers/podman/blob/main/go.mod#L16 and https://github.com/containers/podman/blob/main/vendor/modules.txt#L174 

I guess someone from the core team has to create a new tag 0.60.3 and includes the commit in it and then we can refer to it in the previous files

Reply:
I let you know once we updated it in podman, likely sometime next week. We generally vendor based of the main branch during development so we do not need a new tag there.

Reply:
Hello @Luap99, any news on the previous point ?

Reply:
Sorry I was on PTO so I forget to update you here. Yes, you can rebase the change is vendored now.

Reply:
I guess I'm missing something here :

I don't see any new release of the common lib : https://github.com/containers/common/releases 
And the dependency seems to still be defined in 0.60.1 : https://github.com/containers/podman/blob/main/go.mod#L16

Reply:
We vendor by commit during dev time on main, we only cut releases of the libs when we cut a new podman.
For example https://github.com/containers/podman/commit/b9fe4097ddb98385af85f9417e2601aac313833c.
You can check in the vendor/ directory your changes are in there.